### ISSUE TITLE
What happens to images that arent referenced in posts?

### ISSUE THREAD

**Issue by elapse2039** [havenweb/haven#96]
Probably an unexpected use case, but in a few scenarios I end up deciding to not use a photo, but had uploaded it to post to see it in the preview.

Do these photos remain forever in a database or are they garbage collected if there are no posts referencing them?

**Comment by mawise** [comment]
Yes, these images currently get orphaned.  I do have plans to periodically garbage collect these orphaned images but it is a lower priority than some other work (like your QoL suggestions, and the ability to migrate your haven from one server to another)
